website/blog/2023-12-06-release-1.6.md

```diff
@@ -0,0 +1,250 @@
+
+The minikube extension allows to setup a Kubernetes cluster locally running in a container! Yeah, you read well, in a container, in a similar fashion than Kind. The advantage is that it's lighter and faster to start. With minikube, one of the advantage, is that you can build your images locally with podman and get them automatically available in your local Kubernetes cluster -  which will speed up your turnarounds when you want to test your application. If you want to learn more about the [following blog post](https://podman-desktop.io/blog/sharing-podman-images-with-kubernetes-cluster).
+
+![Minikube-feature-extension](img/podman-desktop-release-1.6/minikube-feature-extension.png)
+
+
+### Command-Line Tools Configuration: Compose and Kubectl
+Configuring and managing your setup is getting easier with the addition of a new section in the Settings to manage Command-line tools. In Podman Desktop, extensions can bring dependencies and tools that are going to be necessary for working properly. 
+
+Two Command-Line tools are getting registered at the moment Compose and kubectl:
+- Compose-cli is registered by the Compose extension. 
+- kubectl is a dependency needed with Minikube extension.
+
+![cli-tools](img/podman-desktop-release-1.6/cli-tools.png)
+
+From the settings you can see the command-line tools that are installed, you can see the version - and when a new version is available, you'll get a small notification to allow you easily updating that dependency. 
+
+![Compose-Update](img/podman-desktop-release-1.6/compose-update.png)
+
+
+### Kubernetes Contexts Manager
+We are introducing a new screen available from the settings allowing to easily manage your Kubernetes contexts. Podman Desktop was already providing the handy context switcher available from the status bar, but when you get to work with multiple Kubernetes environments, it's not uncommon to ended with a big and long list of Kubernetes contexts.
+
+The new Kubernetes Contexts screen allows you to easily see all your registered Kubernetes contexts. You can use that screen to cleanup your registered contexts, and also to define the context you want to be using by default.
+
+![Kubernetes Contexts List](img/podman-desktop-release-1.6/kubernetes-contexts.png)
+
+
+### Editable Podman Machine 
+A Podman machine is a virtual environment specifically designed to run Podman containers on Mac and Windows. It allows users to manage and operate containerized applications in an isolated and controlled setting. When creating a Podman machine, you configure its settings: memory, CPU(s) and disk Size. 
+
+We've got feedback from some of you, interested by the ability to reconfigured your Podman machine on the go. This is now possible and particularly useful when you started with an environment and need to scale it up base on new needs and containers you would like to run in your Podman environment. 
```

Comment:
'interested by the ability to reconfigured' -> 'interested in the ability to reconfigure'

'base' -> 'based'